resource/lvm/lowlevel/size.go

```diff
@@ -0,0 +1,69 @@
+package lowlevel
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+// FIXME: should be caseless RE
+var pctRE = regexp.MustCompile("^(?i)(\\d+)%(PVS|VG|FREE)$")
```

Comment:
Can you document the regular expressions?  They can be kind of hard to read after the fact even when people know regular expressions.